Sources/StreamChat/Utils/OptionalDecodable.swift

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct OptionalDecodable<Base: Decodable>: Decodable {
```

Comment:
Please add a comment explaining what is this and why/when to use it.

Reply:
Fixed.